design/high-level-reactivity.md

```diff
@@ -59,12 +59,18 @@ Note that there is a GraphQL cache - what this does is decompose the results of
 
 This is clearly very simplified, and you can read about how the Relay cache works on [Huey Petersen's website](http://hueypetersen.com/posts/2015/09/30/quick-look-at-the-relay-store/). The important thing to note is that the cache decomposes query results into a flat structure, and a smart cache will be able to generate a query to update just one list and its tasks, or just the list’s name, as necessary.
 
+// May be we can link Learn GraphQL. It's great, specially for a person who dislike GraphQL.
```

Comment:
Sure, if you want to link that in this doc for informational purposes I'd be happy to accept it. Just file a PR and I'll hit the button.